pages/common/autossh.md

```diff
@@ -0,0 +1,21 @@
+# autossh
+
+> Runs, monitors and restarts SSH connections.
+> Auto-reconnects to keep port forwarding tunnels up. Accepts all ssh flags.
+
+- Open an SSH connection with port forward monitoring and forward a local port to a remote one.
+
+`autossh -M {{monitor_port}} {{ssh_command}}`
+`autossh -M {{monitor_port}} -L {{local_port}}:localhost:{{remote_port}} {{user}}@{{host}}`
```

Comment:
I can split it, especially if the double syntax isn't supported.

My logic was that I wanted to not introduce both autoSSH and SSH flags on the same command line.  Specifically, I wanted to make sure the commands paired with the first example actually let you do something.  If I split it to a second entry, the first example requires intricate knowledge of SSH flags before its useful.